src/control/server/ctl_storage_rpc.go

```diff
@@ -100,6 +100,7 @@ func (c *ControlService) scanBdevs(ctx context.Context, req *ctlpb.ScanNvmeReq)
 		return newScanNvmeResp(req, resp, err)
 	}
 
+	c.log.Debugf("bdevs in cfg so only scan those in cfg")
```

Comment:
suggestion - "scanning only bdevs in config"

Reply:
done